src/consensus/raft/test/logging_stub.h

```diff
@@ -3,7 +3,7 @@
 #pragma once
 
 #include "consensus/raft/raft.h"
-#include "consensus/raft/rafttypes.h"
```

Comment:
That seems like it could become types.h

Reply:
I prefer the duplication of `raft/raft_types.h` and `lua/lua_kv.h` to several files called `types.h` and `kv.h`, slightly easier to navigate the code? And ideally these generic names shrink and get removed eventually. Will discuss tomorrow.